src/css/base/variables/_colors.scss

```diff
@@ -1,102 +1,121 @@
-/* Color names from https://chir.ag/projects/name-that-color */
+:root {
```

Comment:
We are currently defining colors in two separate files: `base/_variables.scss` and `variables/_colors.scss`. To improve maintainability, we should consolidate these into a single file. Additionally, we should remove any unused variables and colors.

Reply:
Agree, just want to make sure I don't break anything right now.